### PR TITLE
Adding yarn examples

### PR DESCRIPTION
**Pull request by eranturgeman**
added 2 folders:
yarn-v1 (for yarn classic)
yarn-v2 (for yarn v2 and yarn berry [v3])

### CODE REVIEW

File: yarn-examples/yarn-v2/package.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
+  "name": "yarn-v2",
```
Comment (yahavi) [inline yarn-examples/yarn-v2/package.json:2]: Let's also rename the name of the directory
```suggestion
  "name": "yarn-berry",
```

**Review by yahavi — CHANGES_REQUESTED** [review]
Please also add README.md. See the npm-example for reference.

File: yarn-examples/yarn-v2/package.json [hunk 2]
```diff
@@ -0,0 +1,13 @@
+{
+  "name": "yarn-berry",
+  "version": "1.0.0",
+  "description": "yarn v2 example project",
```
Comment (sverdlov93) [inline yarn-examples/yarn-v2/package.json:4]: v2?

File: yarn-examples/yarn-berry/README.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+## Running the Example
+'cd' to the root project directory
+
+```console
+Configure Artifactory:
+> jf c add --url=<JFROG_PLATFORM_URL> [credentials flags]
+
+Configure the project's repositories:
+> jf yarn-config --repo-resolve=<YARN_RESOLVE_REPO>
+(or just jf yarn config and follow UI instructions)
```
Comment (sverdlov93) [inline yarn-examples/yarn-berry/README.md:22]:
```suggestion
```

File: yarn-examples/yarn-v1/README.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+## Running the Example
+'cd' to the root project directory
+
+```console
+Configure Artifactory:
+> jf c add --url=<JFROG_PLATFORM_URL> [credentials flags]
+
+Configure the project's repositories:
+> jf yarn-config --repo-resolve=<YARN_RESOLVE_REPO>
+(or just jf yarn config and follow UI instructions)
```
Comment (sverdlov93) [inline yarn-examples/yarn-v1/README.md:22]:
```suggestion
```

File: yarn-examples/yarn-v1/README.md [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+
+Collect environment variables and add them to the build info.
+> jf rt bce my-build 1
+
+Publish the build info to Artifactory.
+> jf rt bp my-build 1
```
Comment (sverdlov93) [inline yarn-examples/yarn-v1/README.md:28]:
```suggestion
// Build the project with yarn
jf yarn install
```

File: yarn-examples/yarn-v1/README.md [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+
+Configure the project's repositories:
+> jf yarn-config --repo-resolve=<YARN_RESOLVE_REPO>
+
+Execute 'audit' scan command.
+> jf audit
+```
+
+### Note
+Yarn classic projects are not fully supported by Artifactory as Yarn Berry projects.
```
Comment (sverdlov93) [inline yarn-examples/yarn-v1/README.md:28]:
```suggestion
```

File: yarn-examples/yarn-berry/README.md [hunk 7]
```diff
@@ -0,0 +1,37 @@
...
+
+Publish the build info to Artifactory.
+> jf rt bp my-build 1
+
+Execute 'audit' scan command.
+> jf audit
+```
+
+### Note
+Artifactory supports all Yarn Berry versions but Build operation is currently not supported for projects with version prior to 2.4.0
```
Comment (sverdlov93) [inline yarn-examples/yarn-berry/README.md:37]:
```suggestion
```

**Pull request merged**